dockerfiles/build_manylinux_x86_64.Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM quay.io/pypa/manylinux_2_28_x86_64@sha256:9042a22d33af2223ff7a3599f236aff1e
 ######## Python setup #######
 # These images come with multiple python versions. We pin one for
 # default use.
-ENV PATH="/opt/python/cp312-cp312/bin:${PATH}"
+ENV PATH="/opt/python/cp310-cp310/bin:${PATH}"
```

Comment:
Note that updating the docker file does not cause this patch to test against it. Also, were still hard coded to use the docker from nod.ai